### PR TITLE
homework#1

### PR DESCRIPTION
**Pull request by senya10**

### CODE REVIEW

File: homework1/js/main.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+let string = 'some test string';
+console.log(string[0]);                   //1я буква строки
+console.log(string.slice(-1));            //последняя буква строки
+
+console.log(string[0].toUpperCase()+string.substring(1));         //1я буква верхний регистр
+console.log(string.slice(0,15)+string.slice(-1).toUpperCase());  //последняя буква верхний регистр
+
+console.log(string.indexOf("string"));        //положение string в строке
+
+console.log(string.indexOf(" ", 5));          //положение 2го пробела в строке
```
Comment (moroz112) [inline homework1/js/main.js]: `string.indexOf(' ', string.indexOf(' ') + 1)`

File: homework1/js/main.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+var randMin = 0;                         //Получить случайное целое число от 0 до X. X - любое произвольное число. 
+var randMax = 20;
+var randInt = (Math.floor(Math.random() * (randMax - randMin + 1)) + randMin); 
+console.log(randInt);
+
+var z = 0.6;                                //Проверить результат вычисления 0.6 + 0.7 - как привести к нормальному виду (1.3)
+var y = 0.7;
+var result = z + y;
+console.log(result);
```
Comment (moroz112) [inline homework1/js/main.js:38]: it's fine but you could use template string `${z}${y}`

File: homework1/js/main.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+let string = 'some test string';
+console.log(string[0]);                   //1я буква строки
+console.log(string.slice(-1));            //последняя буква строки
+
+console.log(string[0].toUpperCase()+string.substring(1));         //1я буква верхний регистр
+console.log(string.slice(0,15)+string.slice(-1).toUpperCase());  //последняя буква верхний регистр
```
Comment (moroz112) [inline homework1/js/main.js]: `string.slice(0, string.length - 2 )` cause you shouldn't be counting by yourself

**Review by moroz112 — APPROVED** [review]
good job. A little bit more attention to task descrition

File: homework1/js/main.js (hunk 3 above)
Comment (senya10) [reply homework1/js/main.js]: вот этот момент только не понял что-то...

**Pull request merged**